src/drivers/boards/px4fmu-v4/px4fmu_spi.c

```diff
@@ -103,7 +103,16 @@ __EXPORT void stm32_spi1select(FAR struct spi_dev_s *dev, enum spi_dev_e devid,
 		px4_arch_gpiowrite(GPIO_SPI_CS_MPU9250, 1);
 		px4_arch_gpiowrite(GPIO_SPI_CS_HMC5983, 1);
 		px4_arch_gpiowrite(GPIO_SPI_CS_MS5611, 1);
-		px4_arch_gpiowrite(GPIO_SPI_CS_ICM_20608_G, !selected);
+		px4_arch_gpiowrite(GPIO_SPI_CS_ICM_2060X_G, !selected);
+		break;
+
+	case PX4_SPIDEV_ICM_20602:
```

Comment:
Correct, the 20608 and 20602 are interchangeable in hardware. We are switching to the 20602 because the 20608 is almost EOL.